cookbooks/travis_build_environment/recipes/mysql.rb

```diff
@@ -36,15 +36,30 @@
   execute "echo 'mysql-server-5.6 mysql-server/#{selection} password ' | debconf-set-selections"
 end
 
-package %w[
+mysql_version = 5.6
+mysql_pkgs =  %w[
   libmysqlclient-dev
   libmysqlclient18
   mysql-client-5.6
   mysql-client-core-5.6
   mysql-common-5.6
   mysql-server-5.6
   mysql-server-core-5.6
-] do
+]
+if node['kernel']['machine'] == 'ppc64le' && node['lsb']['codename'] == 'xenial'
```

Comment:
I think this condition can omit the `kernel.machine` check since we'd likely want 5.7 on xenial everywhere.